Sources/Valet/SecureEnclaveAccessControl.swift

```diff
@@ -97,9 +97,9 @@ public enum SecureEnclaveAccessControl: Int, CustomStringConvertible, Equatable
             }
         }
     }
-    
-    internal static func allValues() -> [SecureEnclaveAccessControl] {
-        if #available(macOS 10.12.1, *) {
+
+    public static var allCases: Self.AllCases {
```

Comment:
We need to manually implement `allCases` since different cases are available depending on the OS.